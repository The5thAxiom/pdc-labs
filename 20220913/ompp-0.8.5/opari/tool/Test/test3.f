      program test3

c$OMP    parallel
c$OMP*   do
      do i=1,4
        write(*,*) "pdo", i
      enddo
!$omp end
!$omp+
!$omp+parallel
!$omp+
!$omp+do
      end
