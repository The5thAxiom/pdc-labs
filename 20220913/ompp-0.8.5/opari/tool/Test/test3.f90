      program test3

      !$omp     parallel     & !test !test !test
            !$omp           sections
            write(*,*) "psection"
      !$omp end              &
            !$omp parallel   &
            !$omp sections

      !$omp parallel sections
            write(*,*) "psection"
      !$omp end              &
            !$omp&parallel   &
            !$omp & sections
      end
