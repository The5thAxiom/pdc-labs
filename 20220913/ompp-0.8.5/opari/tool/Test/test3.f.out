#ifdef _POMP
#  undef _POMP
#endif
#define _POMP 200110

#line 1 "test3.f"
      program test3

      call POMP_Parallel_fork(51)
#line 3 "test3.f"
c$omp    parallel
      call POMP_Parallel_begin(51)
      call POMP_Do_enter(51)
#line 3 "test3.f"
c$omp    do
      do i=1,4
        write(*,*) "pdo", i
      enddo
!$omp end do nowait
      call POMP_Barrier_enter(51)
!$omp barrier
      call POMP_Barrier_exit(51)
      call POMP_Do_exit(51)
      call POMP_Parallel_end(51)
!$omp end parallel
      call POMP_Parallel_join(51)
#line 13 "test3.f"
      end
